vividus-plugin-mobile-app/src/main/java/org/vividus/selenium/mobileapp/MobileAppWebDriverManager.java

```diff
@@ -52,21 +53,28 @@ public MobileAppWebDriverManager(IWebDriverProvider webDriverProvider,
 
     public int getStatusBarSize()
     {
-        return  super.isIOSNativeApp() ? getIOsStatusBar() : getAndroidStatusBar();
+        return  super.isIOSNativeApp() ? getStatBarHeight() : getAndroidStatusBar();
     }
 
     private int getAndroidStatusBar()
     {
-        HasAndroidDeviceDetails details = getWebDriverProvider().getUnwrapped(HasAndroidDeviceDetails.class);
-        // https://github.com/appium/java-client/commit/9020174c578bed5e03c24b43c2c5ba590f663201
-        Map<String, Map<String, Object>>  systemBars = CommandExecutionHelper.execute(details,
-                AndroidMobileCommandHelper.getSystemBarsCommand());
-        return Optional.ofNullable(systemBars)
-                       .map(b -> b.get("statusBar"))
-                       .map(sb -> sb.get("height"))
-                       .map(Long.class::cast)
-                       .map(Long::intValue)
-                       .orElse(0);
+        try
+        {
+            HasAndroidDeviceDetails details = getWebDriverProvider().getUnwrapped(HasAndroidDeviceDetails.class);
+            // https://github.com/appium/java-client/commit/9020174c578bed5e03c24b43c2c5ba590f663201
+            Map<String, Map<String, Object>>  systemBars = CommandExecutionHelper.execute(details,
+                    AndroidMobileCommandHelper.getSystemBarsCommand());
+            return Optional.ofNullable(systemBars)
+                           .map(b -> b.get("statusBar"))
+                           .map(sb -> sb.get("height"))
+                           .map(Long.class::cast)
+                           .map(Long::intValue)
+                           .orElse(0);
+        }
+        catch (WebDriverException e)
```

Comment:
do we know/understand the root cause?

Reply:
I was not able to find the issue in Appium. We met the issue for android tv (I am not sure if any of the android TVs could have a status bar at all, but from the session caps I saw that statBarHeight appears in session details, and to be on the safe side I added fall-back code to rely on this capability even if in our case it is 0)

Reply:
please add this comment to `catch` block